pkg/v3/flows/conditional.go

```diff
@@ -19,6 +19,8 @@ import (
 const (
 	// This is the ticker interval for sampling conditional flow
 	SamplingConditionInterval = 3 * time.Second
+	// Maximum number of upkeeps to be sampled in every round
+	MaxSampledConditionals = 300
```

Comment:
If this isn't used outside this package I would lower case it so that it's not exported

Reply:
agree, but currently i would prefer to keep it consistent with other consts in package (which are also used just within the package but capitalised)